### PR TITLE
Rename runtime::TensorView to runtime::Tensor

### PR DESCRIPTION
**Pull request by rkimballn1**
Also renamed runtime::HostTensorView to runtime::HostTensor

### CODE REVIEW

File: src/tools/nbench/benchmark.cpp [hunk 1]
```diff
@@ -164,26 +163,26 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
     cout.imbue(locale(""));
     cout << "compile time: " << timer.get_milliseconds() << "ms" << endl;
 
-    vector<shared_ptr<runtime::HostTensorView>> arg_data;
-    vector<shared_ptr<runtime::TensorView>> args;
+    vector < shared_ptr < runtime::HostTensorrg_data;
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation failed with:
```
src/tools/nbench/benchmark.cpp:166:36: error: ‘HostTensorrg_data’ is not a member of ‘ngraph::runtime’
     vector < shared_ptr < runtime::HostTensorrg_data;
                                    ^~~~~~~~~~~~~~~~~
```

File: src/tools/nbench/benchmark.cpp [hunk 2]
```diff
@@ -164,26 +163,26 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
...
+    vector<shared_ptr<runtime::Tensor>> args;
     vector<bool> args_cacheable;
     for (shared_ptr<op::Parameter> param : f->get_parameters())
     {
         auto tensor = backend->create_tensor(param->get_element_type(), param->get_shape());
         auto tensor_data =
-            make_shared<runtime::HostTensorView>(param->get_element_type(), param->get_shape());
+            make_shared<runtime::HostTensorram->get_element_type(), param->get_shape());
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation error here

File: src/tools/nbench/benchmark.cpp [hunk 3]
```diff
@@ -164,26 +163,26 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
...
         random_init(tensor);
         args.push_back(tensor);
         arg_data.push_back(tensor_data);
         args_cacheable.push_back(param->get_cacheable());
     }
-    vector<shared_ptr<runtime::HostTensorView>> result_data;
-    vector<shared_ptr<runtime::TensorView>> results;
+    vector < shared_ptr < runtime::HostTensoresult_data;
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation error here

File: src/tools/nbench/benchmark.cpp [hunk 4]
```diff
@@ -164,26 +163,26 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
...
+    vector<shared_ptr<runtime::Tensor>> results;
     for (shared_ptr<Node> out : f->get_results())
     {
         auto result = backend->create_tensor(out->get_element_type(), out->get_shape());
         auto tensor_data =
-            make_shared<runtime::HostTensorView>(out->get_element_type(), out->get_shape());
+            make_shared<runtime::HostTensort->get_element_type(), out->get_shape());
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation error here

File: src/tools/nbench/benchmark.cpp [hunk 5]
```diff
@@ -212,10 +211,10 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
         {
             for (size_t arg_index = 0; arg_index < args.size(); arg_index++)
             {
-                const shared_ptr<runtime::TensorView>& arg = args[arg_index];
+                const shared_ptr<runtime::Tensor>& arg = args[arg_index];
                 if (arg->get_stale())
                 {
-                    const shared_ptr<runtime::HostTensorView>& data = arg_data[arg_index];
+                    const shared_ptr < runtime::HostTensorata = arg_data[arg_index];
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation error here

File: src/tools/nbench/benchmark.cpp [hunk 6]
```diff
@@ -227,8 +226,8 @@ vector<runtime::PerformanceCounter> run_benchmark(shared_ptr<Function> f,
         {
             for (size_t result_index = 0; result_index < results.size(); result_index++)
             {
-                const shared_ptr<runtime::HostTensorView>& data = result_data[result_index];
-                const shared_ptr<runtime::TensorView>& result = results[result_index];
+                const shared_ptr < runtime::HostTensorata = result_data[result_index];
```
Comment (shssf) [inline src/tools/nbench/benchmark.cpp]: Compilation error here

File: src/ngraph/descriptor/tensor.hpp [hunk 7]
```diff
@@ -68,7 +68,7 @@ namespace ngraph
             size_t m_pool_offset{0};
         };
 
-        using TensorView = Tensor;
+        using Tensor = Tensor;
```
[the remark this answers is not in the document]
Comment (rkimballn1) [reply src/ngraph/descriptor/tensor.hpp]: no

**Comment by rkimballn1** [comment]
@shssf changing things like removing ngraph:: is nice but it should be done in a separate PR just for that. This PR is only to rename TensorView to Tensor, nothing more.

**Pull request merged**